themes/stylebook-theme/src/css/custom/_challenge_web_content.scss

```diff
@@ -0,0 +1,15 @@
+.challenge-content-title {
+    margin-bottom: 4.375rem;
+}
+
```

Comment:
Overall, the data is displayed as expected but for some reason, locally, the rendered web content does not look like what we have on [Figma](https://www.figma.com/file/jjwYdrZTrBS32BGWxhju65/Frontend-Training?node-id=0%3A1):
Current result:

![image](https://user-images.githubusercontent.com/26236991/186021539-1a2ee656-87a0-4d0c-b905-1523f07f98ac.png)

Expected result:

![image](https://user-images.githubusercontent.com/26236991/186021614-4dd08704-4aea-4fa5-ba7e-b6bf8921241a.png)